### PR TITLE
Micro refactoring

### PR DESCRIPTION
**Pull request by Victor-van-Duijnen**
1. Remove SQLExpress DB from appsettings infrastructure (we had both Dev' and Express connection strings which is confusing)
2. Fix small typo in naming
3. Centralized and refactor semi-broken validation for the EditProfileCommand

### CODE REVIEW

**Comment by Victor-van-Duijnen** [comment]
I have only put you @maxime-poulain, it's the very small refactoring for the edit profile command validator (I have also fixed a typo and useless reference at the same time)

File: src/UserAdmin/src/Smart.FA.Catalog.Application/Extensions/FluentValidation/OnStringRuleBuilderExtensions.cs [hunk 1]
```diff
@@ -12,9 +12,10 @@ public static class OnStringRuleBuilderExtensions
     /// <typeparam name="T">Type of the object being validated.</typeparam>
     /// <param name="ruleBuilder">The <see cref="IRuleBuilder{T,TProperty}" /> on which is applied the rule.</param>
     /// <returns>The source builder source, namely <paramref name="ruleBuilder" />.</returns>
-    public static IRuleBuilder<T, string?> ValidEmail<T>(this IRuleBuilder<T, string?> ruleBuilder)
+    public static IRuleBuilder<T, string?> ValidEmail<T>(this IRuleBuilderInitial<T, string?> ruleBuilder)
```
Comment (maxime-poulain) [inline src/UserAdmin/src/Smart.FA.Catalog.Application/Extensions/FluentValidation/OnStringRuleBuilderExtensions.cs:15]: Hello,

i will rewrite this to make it better and not depenend on an IRuleBiulderInitial

File: src/UserAdmin/src/Smart.FA.Catalog.Application/Smart.FA.Catalog.Application.csproj [hunk 2]
```diff
@@ -15,6 +15,7 @@
     <PackageReference Include="MediatR.Extensions.Microsoft.DependencyInjection" Version="10.0.1" />
     <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="6.0.0" />
     <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="6.0.0" />
+    <PackageReference Include="Mime-Detective" Version="22.7.19" />
```
Comment (maxime-poulain) [inline src/UserAdmin/src/Smart.FA.Catalog.Application/Smart.FA.Catalog.Application.csproj:18]: You could remove from the web project the same nuget as web depends on this project.
I noticed also we track "Mime-Detective.Definitions.Condensed" we can drop that one ?

File: src/UserAdmin/src/Smart.FA.Catalog.Web/Extensions/Middlewares/ProxyHeaderMiddleware.cs [hunk 3]
```diff
@@ -1,5 +1,8 @@
 using System.Security.Principal;
+using System.Threading.Tasks;
 using MediatR;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Http;
```
Comment (maxime-poulain) [inline src/UserAdmin/src/Smart.FA.Catalog.Web/Extensions/Middlewares/ProxyHeaderMiddleware.cs:5]: Those using can be removed.

File: src/UserAdmin/src/Smart.FA.Catalog.Application/UseCases/Commands/EditProfileCommand.cs [hunk 4]
```diff
@@ -162,5 +173,26 @@ public EditProfileCommandValidator()
 
         RuleFor(command => command.Email)
             .ValidEmail();
+        When(request => request.ProfilePicture is not null,
```
Comment (maxime-poulain) [inline src/UserAdmin/src/Smart.FA.Catalog.Application/UseCases/Commands/EditProfileCommand.cs:176]: Could you add breakline between rules, I find it easier tor read.

**Review by maxime-poulain — CHANGES_REQUESTED** [review]

File: src/UserAdmin/src/Smart.FA.Catalog.Application/Extensions/FluentValidation/OnStringRuleBuilderExtensions.cs (hunk 1 above)
Comment (Victor-van-Duijnen) [reply src/UserAdmin/src/Smart.FA.Catalog.Application/Extensions/FluentValidation/OnStringRuleBuilderExtensions.cs:15]: Ok but please, start from this PR and commit the change here too!

File: src/UserAdmin/src/Smart.FA.Catalog.Application/Smart.FA.Catalog.Application.csproj (hunk 2 above)
Comment (Victor-van-Duijnen) [reply src/UserAdmin/src/Smart.FA.Catalog.Application/Smart.FA.Catalog.Application.csproj:18]: Strangely both stuff I had done before, re-pushing it now

**Review by maxime-poulain — APPROVED** [review]

**Pull request merged**